### ISSUE TITLE
Use maxCells instead of maxValues

### ISSUE THREAD

**Issue by likp** [rOpenGov/pxweb#268]
The limit of how many datapoints that can be download on a single request is given by the maxCells given by the config endpoint. 

Currently the code is using maxValues. This setting limits the number of values that is return i metadata calls, use the maxCells instead for proper behaivor.
 
See https://github.com/rOpenGov/pxweb/blob/75a4fe43105cc2032595f3d563a3dba75b206ddc/R/pxweb_add_config.R#L51

**Comment by MansMeg** [comment]
Ok. This has been working for a while so I guess this has come with a new version of pxweb? Do you know?

**Comment by likp** [comment]
It's nothing new, but usually we've set both to the same (or nearly the same) value. This time maxValue was significantly lower than maxCell which led to far more calls than necessary. A solution for us was to raise maxValues.

**Comment by MansMeg** [comment]
Alright. Is there any documentation I could refer to? I remember when I set this up (probably back in 2013ish), `maxValue` was the only value returned by the API ?config. Since there are so many users and this is one of the core functionalities, I don't want to mess with it unless I know that changing this will not break it.

**Comment by MansMeg** [comment]
Of special interest is which to choose if both occur. I.e. which one is the true limit?

**Comment by likp** [comment]
The answer to ?config seems to be undocumented so I had to look at old code.
You are correct that maxCells were added in 2019 and this value is the actual limit on the size of data retrivials that can be made.
maxValues has been there from the start but has always described the maximum number of values that will be returned for a variable during a metadata extraction.
Therefore I would always check if maxCells is present in the ?config response and use this if so. If it wasn't there, I would default to 100,000, which is the default value for maxCells. As it is most likely that this has not changed.
